app/gui/view/src/root.rs

```diff
@@ -0,0 +1,152 @@
+//! Root View of the IDE.
+//!
+//! The main entry point to the IDE which can display either Welcome Screen or Project View.
+//! Initially displays Welcome Screen. As soon as user selects a project to open, Root View
+//! switches state to Project View and stays in this state till the application quit.
+// TODO [vitvakatu]: at the moment Root View is not used as described, but it will be used in the
+// following tasks.
```

Comment:
what does it mean "at the moment" and what are "following tasks"? How would we guarantee that this comment will be updated when the "following tasks" will be implemented? Also, it's very hard to check this code because its hard to understand what Root View now does, as it is not doing what doc describes.

Reply:
TODO comment removed, documentation also reflects current behavior